lib/views/logInView.dart

```diff
@@ -0,0 +1,167 @@
+                          floatingLabelAlignment: FloatingLabelAlignment.center,
+                          enabledBorder: InputBorder.none,
+                        )
+                    ),
+                  )
+              ),
+              Center(
+                child: Container(
+                  height: 50.0,
+                  width: 303.0,
+                  child: Align(
+                    alignment: AlignmentDirectional.bottomEnd,
+                    child: TextButton(
+                      onPressed: () {},
+                      style: TextButton.styleFrom(
+                        fixedSize: const Size(110, 14),
+                      ),
+                      child: Text(
+                          '忘記密碼',
+                          style: TextStyle(
+                            color: 'F5F5DC'.toColor(),
+                            fontSize: 14.0,
+                            fontFamily: 'Serif',
+                          )
+                      ),
+                    ),
+                  ),
+                ),
+              ),
+              SizedBox(height: deviceHeight * 0.0379),
+              Center(
+                  child: Row(
+                      mainAxisSize: MainAxisSize.min,
+                      children: <Widget>[
+                        TextButton(
+                            onPressed: () {},
+                            style: TextButton.styleFrom(
+                              fixedSize: const Size(110, 45),
+                            ),
+                            child: Text(
+                                '登入',
+                                style: TextStyle(
+                                  fontSize: 24.0,
+                                  fontFamily: 'Serif',
+                                  color: 'F5F5DC'.toColor(),
+                                )
+                            )
+                        ),
+                        Text(
+                            '/',
+                            style: TextStyle(
+                              fontSize: 24.0,
+                              fontFamily: 'Serif',
+                              color: 'F5F5DC'.toColor(),
+                            )
+                        ),
+                        TextButton(
+                            onPressed: () {},
+                            style: TextButton.styleFrom(
+                              fixedSize: const Size(110, 45),
+                            ),
+                            child: Text(
+                                '註冊',
+                                style: TextStyle(
+                                  fontSize: 24.0,
+                                  fontFamily: 'Serif',
+                                  color: 'F5F5DC'.toColor(),
+                                )
+                            )
+                        )
+                      ])
+              ),
+            ]
```

Comment:
![image](https://github.com/dlion168/learning-to-read-chinese/assets/75151679/f8c9ba31-e27e-4a4f-8074-5523acd8cf65)
overflow error

Reply:
fixed! thank u:)